File: BotsSDK/widgetsdk/src/main/java/com/kore/ai/widgetsdk/models/BotInfoModel.java
package com.kore.ai.widgetsdk.models;

import java.util.HashMap;

/**
 * Created by Pradeep Mahato on 06-Jun-16.
 * Copyright (c) 2014 Kore Inc. All rights reserved.
 */
public class BotInfoModel {
    public String chatBot;
    public String taskBotId;
    public String client = "Android";

    public String getChannelClient() {
        return client;
    }

    public void setChannelClient(String channelClient) {
        this.client = channelClient;
    }


    public HashMap<Object,Object> customData;


    public BotInfoModel(String chatBot, String taskBotId,HashMap customData) {
        this.chatBot = chatBot;
        this.taskBotId = taskBotId;
        this.customData = customData;
    }

    public void setChatBot(String chatBot) {
        this.chatBot = chatBot;
    }

    public void setTaskBotId(String taskBotId) {
        this.taskBotId = taskBotId;
    }
}

File: BotSdk/botclient/src/main/java/com/kore/model/BotInfoModel.kt
package com.kore.model

import com.google.gson.annotations.SerializedName

/**
 * Copyright (c) 2023 Kore Inc. All rights reserved.
 */
data class BotInfoModel(
    @SerializedName("chatBot")
    val botName: String,
    @SerializedName("taskBotId")
    val botId: String,
    val customData: Map<String, String?>?,
    val channelClient: String = "Android"
)